assets/lokomotive-kubernetes/bootkube/resources/charts/kubelet.yaml

```diff
@@ -1,3 +1,4 @@
 image: ${hyperkube_image}
 clusterDNS: ${cluster_dns_service_ip}
 clusterDomain: ${cluster_domain_suffix}
+disable: ${disable}
```

Comment:
I don't think it make sense. I suggest we don't install the chart at all.

Reply:
With the `count = 0` for those 2 resources: https://github.com/kinvolk/lokomotive/blob/master/assets/lokomotive-kubernetes/bootkube/assets.tf#L100-L114.

Reply:
Do you mean at the terraform level we control the installtion of the chart?

Reply:
done